src/App.tsx

```diff
@@ -1,14 +1,33 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { FilteredTodos } from './utilities/FilteredTodos';
+
+export type FilterTodoBy = 'all' | 'active' | 'completed';
```

Comment:
Minor - but probably you should move it to `TodoFilter`

Reply:
Hello, replace export type FilterTodoBy = 'all' | 'active' | 'completed'; to TodoFilter